html/scss/navbar.scss

```diff
@@ -0,0 +1,20 @@
+.navbar {
+  position: fixed;
+  width: 100%;
+  height: $nav-height;
+  background: $white;
+  box-shadow: 0 5px 20px $grey;
+  z-index: 999;
+}
+
+.nav-wolox {
+  position: absolute;
+  height: $nav-height;
+  width: 100%;
+  background: url('../assets/wolox_logo.svg') no-repeat;
```

Comment:
you should use an image in the html for this case. this is part of the content and not a background or decoration.